Lib/test/test_readline.py

```diff
@@ -210,8 +210,10 @@ def display(substitution, matches, longest_match_length):
         self.assertIn(b"result " + expected + b"\r\n", output)
         self.assertIn(b"history " + expected + b"\r\n", output)
 
-    @unittest.skipIf(is_editline,
-                     "editline history size configuration is broken")
+    # History size was added in readline 6.0
+    # See https://cnswww.cns.cwru.edu/php/chet/readline/CHANGES
+    @unittest.skipIf(getattr(readline, "_READLINE_VERSION", 0x00) < 0x600,
```

Comment:
I think we should fix the old code not to assume private attributes, same as @haypo patch for printing readline version. 

For this patch I'll drop the fallback.